examples/basics/scene/sphere.py

```diff
@@ -11,14 +11,27 @@
 import sys
 
 from vispy import scene
-canvas = scene.SceneCanvas(keys='interactive', size=(800, 600), show=True)
+from vispy.visuals.transforms import STTransform
+
+canvas = scene.SceneCanvas(keys='interactive', bgcolor='white',
+                           size=(800, 600), show=True)
 
 view = canvas.central_widget.add_view()
 view.camera = 'arcball'
-view.padding = 100
 
-sphere = scene.visuals.Sphere(radius=1, parent=view.scene,
-                              edge_color='black')
+sphere1 = scene.visuals.Sphere(radius=1, method='latlon', parent=view.scene,
+                               edge_color='black')
+
+sphere2 = scene.visuals.Sphere(radius=1, method='ico', parent=view.scene,
+                               edge_color='black')
+
+sphere3 = scene.visuals.Sphere(radius=1, method='cube', parent=view.scene,
+                               edge_color='black')
```

Comment:
the `cube` has more triangles than the others, it's probably worth changing the params to make it look more similar